lib/rspec/core/configuration.rb

```diff
@@ -1412,13 +1415,32 @@ def self.delegate_to_ordering_manager(*methods)
       #     end
       #   end
       #
-      #   describe MyClass, :order => :reverse do
+      #   RSpec.describe 'MyClass', :order => :reverse do
       #     # ...
       #   end
       #
       # @note Pass the symbol `:global` to set the ordering strategy that
       #   will be used to order the top-level example groups and any example
       #   groups that do not have declared `:order` metadata.
+      #
+      # @example
+      #   RSpec.configure do |rspec|
+      #     rspec.register_ordering :global do |examples|
+      #       acceptance, other = examples.partition do |example|
+      #         example.metadata[:type] == :acceptance
```

Comment:
This API is for ordering examples, you are given examples and are free to order them as you wish. This PR is about improving the documentation of the fact you may order examples as you wish.

If you wish to expand upon the available ordering strategies may I suggest you work on an extension gem that adds whatever complex ordering you desire, if any of those show enough public adoption then we may consider bringing them into rspec-core, however for now please can you limit discussion on this topic to useful feedback about the documentation. I am not going to expand this example further at this time.